### PR TITLE
day03 hw

### PR DESCRIPTION
**Pull request by charleytran**
Need to do the bonuses. need time.

### CODE REVIEW

File: charley_tran/week_01/arrays.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	if (wordToGuess[x] === guess) {
+		console.log("Congratulations you found letter " + guess);
+		currentGuessedLetters[x] = guess;
+		console.log(currentGuessedLetters);
+	}   
+	}		checkComplete();
+		checkWinner();
+}
+
+
```
Comment (puyo) [inline charley_tran/week_01/arrays.html:43]: I really like your use of functions and your choice of variable names. It made it easier to understand this solution than some of the others.

**Comment by charzy** [comment]
Hi Greg
Thanks for the feedback. I had difficulty in understand the question as it's not very clear; I had to redefine the second array in much simpler term (Initially I thought the second array was to cache the guessed letters).
I have a question however that's been bugging me. With the following line of code,

if (currentGuessedLetters===wordToGuess) {
+		return console.log("Congratulations we have a winner!");
+	}
+}

I was using this if statement to execute if the elements in the two arrays are identical. However this didn't seem to work hence I had to create an expression to reduce the

**Comment by charleytran** [comment]
Thanks for the feedback Greg!
Defining the arrays were driving me nuts so I had to name them simply.

**Comment by ButenkoT** [comment]
You started to do more commits and comments! That's great!

**Pull request merged**